src/TestLogger/Extensions/XunitTestAdapter.cs

```diff
@@ -46,6 +46,14 @@ public class XunitTestAdapter : ITestAdapter
                     result.Messages.Add(new TestResultMessage("skipReason", skipReason));
                 }
 
+                string displayName = result.Result.DisplayName;
+
+                // Add parameters for theories.
+                if (string.IsNullOrWhiteSpace(displayName) == false && displayName.Contains("("))
```

Comment:
Minor: we can use `IndexOf("(")` and see if it is `< 0` instead of `Contains` and `IndexOf`.